hedera-node/hedera-app-spi/src/main/java/com/hedera/node/app/spi/throttle/ThrottleAdviser.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (C) 2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hedera.node.app.spi.throttle;
+
+import com.hedera.hapi.node.base.HederaFunctionality;
+
```

Comment:
Perhaps this deserves a javadoc at the interface level?

Perhaps especially, since, at the interface level, the comment at L36-37 makes no sense since it isn't at all clear _what the child transactions are it is talking about_.  Those are injected into the implementation's constructor, and don't appear here at all!
* (Made more sense when this was part of the `HandleContext` and so much more was going on over there.)

Reply:
Oh, an oversight. I will add interface level documentation in a follow-up PR.